app/decorators/course_decorator.rb

```diff
@@ -0,0 +1,7 @@
+class CourseDecorator < ApplicationDecorator
+  delegate_all
+
+  def last_published_at
+    object&.last_published_at&.to_date&.strftime("%-d %B %Y")
```

Comment:
I don't think we should safeguard `object` being nil. i.e. just simply `object.last_published` without the `&`.

Reply:
This was a hold-over since before `last_published_at` was implemented I believe, can be removed now.